tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/date_and_time.py

```diff
@@ -21,3 +29,59 @@ def __init__(self, marionette):
     def select_time_format(self, time_format):
         self.marionette.find_element(*self._24h_selector_locator).tap()
         self.select(time_format)
+
+    def set_automatic_time_update(self, state):
+        """
+        toggles 'Set Time Automatically' flag
+        """
+
+        Wait(self.marionette).until(expected.element_displayed(
+            Wait(self.marionette).until(
+                expected.element_present(*self._autotime_enabled_locator))))
+
+        if not state:
+            if self.is_autotime_enabled is True:
```

Comment:
Aha, I see what you meant.  so you're saying that when I tap the autotime, the tester has to be aware of the state of the autotime switch.  Initially what I meant to do was to enable it regardless of the current state of the autotime, because my goal is have it enabled regardless.  Make sense, that logic should be apparent in the test script.  I'll make the change.